gems/pending/appliance_console/utilities.rb

```diff
@@ -29,23 +29,23 @@ def self.bail_if_db_connections(message)
       end
     end
 
-    def self.db_host_type_database
-      result = AwesomeSpawn.run("bin/rails runner",
-                                :params => ["puts MiqDbConfig.current.options.values_at(:host, :adapter, :database)"],
-                                :chdir  => RAILS_ROOT
-                               )
+    def self.db_host_database_region
+      result = AwesomeSpawn.run(
+        "bin/rails runner",
+        :params => ["puts MiqDbConfig.current.options.values_at(:host, :database), MiqDatabase.my_region_number"],
```

Comment:
Prefer `ActiveRecord::Base.my_region_number`...no need to couple to a specific table.